### DISCUSSION TITLE
Videos randomly unpausing after pause

### DISCUSSION THREAD

**Discussion by SGvagon** [ReVanced/revanced-discussions#1085] [Question]
Hello,

I have issue with ReVanced which randomly unpausing videos after I paused. It maybe doing Spoofing the app, but I like the old Youtube design.

Is this issue fixable with any settings or patch?

I am using the latest version ReVanced app v19.16.39-patches_v4.17.0 . 

I tried post this issue on Reddit, but my question immidiately removed after I sent it

Sorry, this post has been removed by the moderators of r/revancedapp.

I don't know why.

Thank you in advance.

**Comment by oSumAtrIX** [comment]
The issue is not present on 4.17 and was present on older patches version. If you think otherwise follow this:

![image](https://github.com/user-attachments/assets/a9b03ed1-d190-4cf0-b4f7-2c860eb3d050)
